client/my-sites/hosting/main.js

```diff
@@ -43,7 +43,13 @@ const Hosting = ( { translate, isDisabled, canViewAtomicHosting, siteId } ) => {
 			<SidebarNavigation />
 			<FormattedHeader
 				headerText={ translate( 'Hosting Configuration' ) }
-				subHeaderText={ translate( 'Access and manage more advanced settings of your website.' ) }
+				subHeaderText={
+					sftpPhpMyAdminFeaturesEnabled
+						? translate(
+								'Access your website, database, and more advanced settings of your website.'
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.